### DISCUSSION TITLE
Unsubscribe URL goes to blank white page with success message. Also some subscription advise

### DISCUSSION THREAD

**Discussion by dbrown199747** [solspace/craft-freeform#147] [Questions & Answers] [answered]
Hi,

I am implementing a subscription option on my Craft CMS site and I am using Freeform to do that. Currently it's working brilliantly and all my test subscriptions do what I need. The one question I had was that the unsubscribe URL takes the user to a blank white page with a text message on. Is there anyway to keep the user on the site instead of taking them to this page, or even style it in any way? Currently it looks a little unprofessional.

Also, I was wondering if anyone had any advise on the best way to set up and use a subscription on my craft site. I currently only want to show certain content on the site if the user is subscribed. The way I have done it currently is by adding the freeform credit card fields to a sign up form, then I know that if a user is logged in, they would be subscribed. My main issue comes down to unsubscribing, and how to know if a user has unsubscribes, and also if there is a way of automatically deactivating their account once the user unsubscribes (unless anyone has a better way to do it)?

If anyone has any ideas on how I can achieve this, i'd love to hear them.

Many thanks

**Comment by kjmartens** [comment, marked as answer]
@dbrown199747 By subscription, I assume you mean Payment subscriptions? Currently this is not possible, but we'll make a note about it for future consideration to see if we can offer more control over this. :)

As for your other question, there isn't a built-in way to do this right now, as Freeform submissions / payments are not tied to user accounts in any way.
